src/prefect/task_worker.py

```diff
@@ -141,7 +151,7 @@ def current_tasks(self) -> Optional[int]:
     def available_tasks(self) -> Optional[int]:
         return int(self._limiter.available_tokens) if self._limiter else None
 
-    def handle_sigterm(self, signum, frame):
+    def handle_sigterm(self, signum: int, frame: Any):
```

Comment:
```suggestion
    def handle_sigterm(self, signum: int, frame: object):
```

Unused arguments should generally be typed as `object` so the type checker will scream at you when you start using them later (and so remind you to provide proper types).

In typeshed they even provide a type alias called `Unused` for this, which is just `Unused = object`.